atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -60,7 +60,15 @@ def run(self):
 
         target_info = self.xmlrpc.getBuildTarget(self.target)
         self.log.debug('target info: %s', target_info)
+        if not target_info:
+            raise RuntimeError("no such tag: %s" % self.target)
+
         tag_id = target_info['dest_tag']
+        try:
+            assert self.xmlrpc.checkTagPackage(tag_id, component)
```

Comment:
I wanted to protect against exception in ```checkTagPackage``` call - in case the user has insufficient permissions or our brew doesn't have it.

I'll probably revert this to a simple ```if``` check